### PR TITLE
ui: Show the creation time in the ort runs table and run details page

### LINKED ISSUE #656: Show the creation time in the ORT runs table
Currently, the ORT runs table only shows the date when a run was created. Also show the time, because the date alone is not sufficient information when many runs were started on the same day.

image

### PR DESCRIPTION
**Pull request by Etsija**
Use browser language to decide on the exact format. Examples:

English:
Capture_English

Finnish:
Capture_Finnish

Run details, English:
Capture_details_English

### CODE REVIEW

**Comment by sschuberth** [comment]
In the run details, we separate date and time with a comma:

image

For consistency and readability I believe we should do so here, too.

**Comment by Etsija** [comment]
> In the run details, we separate date and time with a comma:
> 
> image
> 
> For consistency and readability I believe we should do so here, too.

We do! Please look at the run list, English example. The code to produce the date-time fields is exactly the same.

**Comment by sschuberth** [comment]
> Resolves #656.

This should go to the message of the commit that concludes the resolution.

**Comment by sschuberth** [comment]
> We do! Please look at the run list, English example. The code to produce the date-time fields is exactly the same.

Oh, sorry. So using a comma or not is also defined by the locale? Weird!

**Comment by Etsija** [comment]
> > We do! Please look at the run list, English example. The code to produce the date-time fields is exactly the same.
> 
> Oh, sorry. So using a comma or not is also defined by the locale? Weird!

Not really, as each country/language uses their own specific way to represent date and time. I could've chosen Hindi as my example as well, I'm sure it would look very different 😄

**Review by sschuberth — APPROVED** [review]

**Comment by sschuberth** [comment]
> Not really, as each country/language uses their own specific way to represent date and time. I could've chosen Hindi as my example as well, I'm sure it would look very different 😄

Ok, let me rephrase 😉 : Weird that Finns don't separate date and time by a comma, esp. as each date and time is separated by dots, making it hard to distinguish visually.

**Review by mmurto — APPROVED** [review]

**Comment by mmurto** [comment]
> > Not really, as each country/language uses their own specific way to represent date and time. I could've chosen Hindi as my example as well, I'm sure it would look very different 😄
> 
> Ok, let me rephrase 😉 : Weird that Finns don't separate date and time by a comma, esp. as each date and time is separated by dots, making it hard to distinguish visually.

There's a space in it, IMO it's the best separator. :D

**Comment by Etsija** [comment]
> > > Not really, as each country/language uses their own specific way to represent date and time. I could've chosen Hindi as my example as well, I'm sure it would look very different 😄
> > 
> > 
> > Ok, let me rephrase 😉 : Weird that Finns don't separate date and time by a comma, esp. as each date and time is separated by dots, making it hard to distinguish visually.
> 
> There's a space in it, IMO it's the best separator. :D

Oh don't get me started on the greatness of a DOUBLE SPACE after each sentence! 😆

**Comment by sschuberth** [comment]
> There's a space in it, IMO it's the best separator. :D

At least you did not mention tabs... 👿
